.github/workflows/redeploy.yml

```diff
@@ -0,0 +1,57 @@
+name: Redeploy
+
+on:
+  workflow_dispatch:
+    inputs:
+      dockerTag:
+        description: 'Docker tag to deploy'
+        required: true
+        type: string
+      branch:
+        description: 'Branch or commit used for configs'
+        required: true
+        type: string
+        default: 'main'
+      environment:
```

Comment:
I find the naming of this input confusing, because `environment` as a term is already used by GitHub.
I.e., we have configured GitHub Environments
* dev-eu-west-1
* dev-us-east-2
* prod-us-west-1
* prod-us-east-2
* prod-us-west-2
* prod-eu-central-1
* prod-ap-southeast-1

The `environment` input variable in this PR is one level higher.
How about calling it `environment_kind`.

The naming rule is then:
```
jobs:
 jobname:
    environment:
      # The GH Actions Environment naming scheme is {dev,prod}-$region
      name: ${{ inputs.environment_kind }}-${{matrix.target_region}}